barretenberg/cpp/src/barretenberg/vm2/simulation/gadgets/calldata_hashing.hpp

```diff
@@ -12,6 +12,7 @@ namespace bb::avm2::simulation {
 
 class CalldataHasher : public CalldataHashingInterface {
   public:
+    // TODO should this be uint16? range check is for 16 bit
```

Comment:
Before merging - I think this is fine (context id appears to be uint32 in most places) but just to double check